### ISSUE TITLE
[Bug] Криво считается % занятого объема виртуальной памяти

### ISSUE THREAD

**Issue by 3bagorion33** [henrypp/memreduct#235]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

3.4

### Windows version

Windows 11

### Steps to reproduce

1. Установить.
2. Открыть.

### Actual behavior

![изображение](https://github.com/user-attachments/assets/d5250f39-d232-4de2-9e99-c630dc79c8b8)

**Comment by henrypp** [comment]
И где криво? Математика с тобой не согласна:

```c
ULONG64 total_length = 33071248179; // 30,8
ULONG64 length = 11059540787; // 10,3

(length / total_length) * 100 // получаются всё те же 66%, можешь скомпилить - проверить!
```

**Comment by henrypp** [comment]
Ну или в калькуляторе, если не умеешь компилить

**Comment by 3bagorion33** [comment]
Да, я умею компилировать. А еще, в отличие от жертв ЕГЭ, я умею считать устно и решать неравенства, потому как учился в постсоветской школе.

Пусть х - занятая часть памяти. Тогда занятый % физической памяти m определяется как 
x/m * 100%
Аналогично для виртуальной памяти M
x/M * 100%

Сравним два эти значения

x/m * 100% ? x/M * 100%

Одно из свойств неравенства гласит, что при умножении обоих частей на положительное число знак неравенства не меняется. Домножим на 100%/x

1/m ? 1/M

Даже если не вдаваться в терминологию и оперировать только числами на экране, мы получим
m = 15.2GB
M = 30.8GB

Другое свойство неравенств гласит, что если a < b, то 1/a > 1/b

Очевидно, что m < M, поскольку 15,2ГБ < 30,8ГБ. Значит
1/m > 1/M =>
x/m * 100% > x/M * 100%

Таким образом, % отображаемой занятой физической памяти должен быть больше % отображаемой занятой виртуальной памяти. А 38% > 66% - неверно.

Теперь вернемся к тому, что такое виртуальная память. Согласно википедии https://en.wikipedia.org/wiki/Memory_virtualization

> Virtual memory systems abstract between physical RAM and virtual addresses, assigning virtual memory addresses both to physical RAM and to disk-based storage, expanding addressable memory, but at the cost of speed.

Таким образом, здесь явно сказано, что физическая память является **частью** виртуальной памяти. А то, что часть меньше целого - это база, которую дают, наверное, еще в детском садике.
